dev/how-to/upgrade/tidb-in-kubernetes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Rolling Update TiDB in Kubernetes
+category: how-to
+---
+
+# Rolling Update TiDB in Kubernetes
+
+When you perform a rolling update for a TiDB cluster in Kubernetes, the Pod is shut down and recreated with the new image or/and configuration serially in the order of PD, TiKV, TiDB. Under the high available deployment topology (minimum requirements: PD \* 3, TiKV \* 3, TiDB \* 2), rolling updating PD and TiKV servers does not impact the running clients.
+
+For clients that can retry stale connections, rolling updating TiDB servers does not impact the running clients neither. However, for clients that **can not** retry stale connections, rolling updating TiDB servers will close the client connections and cause the request to fail. We recommend to add connection retry ability for the clients or to perform rolling update for TiDB servers in idle time.
+
+## Upgrade TiDB cluster version
+
+1. Change `image` of PD, TiKV and TiDB to different image versions in `values.yaml`.
+2. Run `helm upgrade` command：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm upgrade ${releaseName} pingcap/tidb-cluster -f values.yaml --version=v1.0.0-beta.3
+    ```
+
+3. Witness the upgrade progress：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    watch kubectl -n ${namespace} get pod -o wide
+    ```
+
+## Change TiDB cluster Configuration
+
+By default, the change of configuration files is not applied to the TiDB cluster automatcially. The servers load the new configuration file when being restarted.
+
+You can enable rolling update on configuration change by setting `enableConfigMapRollout` to `true` in the `values.yaml`. The steps are as follows:
```

Comment:
Suggest setting `enableConfigMapRollout` to true by default in values.yaml, it's not reasonable to request customer to delete pod manually after config change and helm upgrade.